src/Build/BackEnd/Components/RequestBuilder/IntrinsicTasks/ItemGroupIntrinsicTask.cs

```diff
@@ -433,7 +433,11 @@ ISet<string> removeMetadata
                     string[] includeSplitFiles = EngineFileUtilities.GetFileListEscaped(
                         Project.Directory,
                         includeSplit,
-                        excludes);
+                        excludes,
+                        loggingMechanism: LoggingContext,
+                        includeLocation: originalItem.IncludeLocation,
+                        excludeLocation: originalItem.ExcludeLocation,
+                        disableExcludeDriveEnumerationWarning: true);
```

Comment:
It's set so in EngineFileUtilities, if there is an Exclude file spec detected in FileMatcher and the first GetFileListEscaped call is used from ItemGroupIntrinsicTask, it does not log a warning. It will wait until the second GetFileListEscaped call is made with the list of excludes before logging a warning (this is so the proper attributes and elements will be used in the warning - otherwise, it is more difficult to detect the root cause and placement of the filespec in the project file).